### PR TITLE
Switch from allpars to tbl

### PR DESCRIPTION
**Pull request by dmbates**
- Use the `tbl` property of a parametricbootstrap value instead of the `allpars` property when feasible
    + the `allpars` property is a `long-format` column table of `iter`, `type`, `group`, `names` and `value`
    + the advantage of having all the parameter values in one long column is that the table can be passed to a graphics call using either `group` or `names` to select color or panel.
    + a disadvantage is that it is usually necessary to filter the column table by `type` and perhaps values in other columns to get the values that should be plotted
    + another disadvantage is that it is difficult to review the values quickly because the order of the rows is by iteration number so the values of one parameter are not adjacent
    + the `tbl` property is a `wide` table where each parameter corresponds to a column
    + this orientation make extraction of the values of a particular parameter trivial
    + the column names are derived from parameter type and a number within type
    + the plan is to have a function that translates from parameter type and number to the longer names
    + comparative plots can be created with the `wide` syntax in `AlgebraOfGraphics`.  This is not as convenient as the default specification but is accessible.

- Worked on reducing the number of packages attached in a `.qmd` file with `using`.
    + the `ProgressMeter` package is only ever used for one call with a fully qualified name so it can be attached with `import`
    + `using SMLP2023` can be elminated in favor of `using SMLP2023: dataset`

- Fixed some syntax of literature citations and some spelling mistakes

- use `#| output: false` instead of ending code blocks with a semicolon

- eliminated some vestigial `begin` and `end` lines from Pluto notebooks that were converted to Quarto

- eliminated problematic smoother plots of the binary `acc` response versus word length in `largesampledesigned.qmd`
    + these plots could take a very long time with recent versions of the `loess` package and were not very informative

### CODE REVIEW

**Comment by kliegl** [comment]
@dmbates This looks all very good to me. Thanks for doing all this work!

**Comment by dmbates** [comment]
Thank you for merging the main branch, @palday.  I remembered to check out the main branch before creating this branch but forgot to `git pull` after checking it out.  This git stuff is an "old doug, new tricks" situation for me.

**Comment by palday** [comment]
@dmbates if it's any consolation, I think I messed up a few spots and you might have to update a few things when you call `quarto render`.

File: contrasts_fggk21.qmd [hunk 1]
```diff
@@ -51,7 +49,7 @@ using SMLP2023: dataset
 using Statistics
 using StatsBase
 
-ProgressMeter.ijulia_behavior(:clear);
+ProgressMeter.ijulia_behavior(:clear)
```
Comment (palday) [inline contrasts_fggk21.qmd:52]:
```suggestion
ProgressMeter.ijulia_behavior(:clear);
```

**Pull request merged**